l5kit/l5kit/visualization/utils.py

```diff
@@ -3,14 +3,15 @@
 import cv2
 import numpy as np
 
-from l5kit.geometry import transform_points
+from l5kit.geometry import transform_point, transform_points
```

Comment:
Very nit: do we really need 2 methods for this in the geometry module? Might be neater to have just the transform_points and send in array of 1 point. I'm just worried if you make a typo in 1 letter, you may get a very different behaviour I assume.